examples/getting_started/react/README.md

```diff
@@ -0,0 +1,20 @@
+# Getting started
+
+```
+yarn install
+```
+
+## Setting up a server
+
+To get started, you will need to set up a GraphQL server. 
```

Comment:
So server will not be included? Without server, we will not be able to show conflicts

Reply:
Going to add a server that can be used by this example and React Native too